koku/providers/test/oci/test_oci_provider.py

```diff
@@ -97,6 +106,19 @@ def test_check_cost_report_access_client_error(self, mock_storage_client):
         with self.assertRaises(ValidationError):
             provider_interface.cost_usage_source_is_reachable(FAKE.md5(), data_source)
 
+    @patch("providers.oci.provider.storage_client.ObjectStorageClient")
+    def test_check_cost_report_access_parse_error(self, mock_storage_client):
+        """Test_check_cost_report_access error."""
+        mock_storage_client.return_value = mock_OCI_Parse_client
```

Comment:
I'm pretty sure you could do:
```
mock_storage_client.return_value.list_objects.side_effect = LocationParseError('test exception')
```

which would avoid having to define that mock class.